frontend/src/pages/index.tsx

```diff
@@ -1,27 +1,6 @@
-import React, { FC } from "react";
-import { QueryCache, ReactQueryCacheProvider } from "react-query";
-import { ReactQueryDevtools } from "react-query-devtools";
-import { isSSR } from "src/common/utils/isSSR";
-import AppContainer from "src/components/AppContainer";
-import CookieBanner from "src/components/CookieBanner";
-import Layout from "../components/Layout";
-import SEO from "../components/seo";
+import React from "react";
+import Homepage from "src/views/Homepage";
 
-const queryCache = new QueryCache();
+const Page = () => <Homepage />;
```

Comment:
Index page doesn't render the global layout anymore, since that's `_app.tsx` and `_document.tsx`'s jobs now